src/Maestro/Client/src/MaestroApi.cs

```diff
@@ -1,32 +1,36 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Azure.Core;
 using Newtonsoft.Json.Linq;
 using System;
 using System.IO;
 using System.Net;
 using System.Net.Http;
 using System.Net.Sockets;
 
+#nullable enable
 namespace Microsoft.DotNet.Maestro.Client
 {
     internal partial class MaestroApiResponseClassifier
     {
-        public override bool IsRetriableException(Exception exception)
-        {
-            return base.IsRetriableException(exception) ||
-                exception is OperationCanceledException ||
-                exception is HttpRequestException ||
-                exception is RestApiException raex && raex.Response.Status >= 500 && raex.Response.Status <= 599 ||
-                exception is IOException ||
-                exception is SocketException;
-        }
+        public override bool IsRetriableException(Exception exception) =>
+            base.IsRetriableException(exception)
+                || exception is OperationCanceledException
+                || exception is HttpRequestException
+                || (exception is RestApiException raex && raex.Response.Status >= 500 && raex.Response.Status <= 599)
+                || exception is IOException
+                || exception is SocketException;
     }
 
     public partial class MaestroApi
     {
-        //Special error handler to consumes the generated MaestroApi code. If this method returns without throwing a specific exception
-        //then a generic RestApiException is thrown.
+        public const string ProductionBuildAssetRegistryBaseUri = "https://maestro.dot.net/";
```

Comment:
is there a way to add a new parameter to `src/Maestro/Maestro.Web/.config/settings.Production.json` (+staging) and read from it?

Reply:
It would have to be some magic build step as this project should not depend on the `Web` project. Not sure how it would work exactly though.

Reply:
ah. ok I just noticed the config file is in different project.

Reply:
Yeah, the web services have environments but `darc` can call both environments, so it needs to have information about both somewhere.